server/service/src/sync/translations/requisition.rs

```diff
@@ -211,6 +211,18 @@ impl SyncTranslation for RequisitionTranslation {
             ),
         };
 
+        // TODO: Delete when soft delete for master list is implemented
+        let program_id = if let Some(program_id) = data.programID {
+            let program = ProgramRowRepository::new(conn).find_one_by_id(&program_id)?;
+
+            match program {
+                Some(program) => Some(program.id),
+                None => Some(MISSING_PROGRAM.to_string()),
```

Comment:
hmm this is not quite what I've envisioned, this would mean we would loose the previous reference, I think it's ok for now since this is not likely to happen, but thought we would create new program and new master list based on the missing master list id in the requisition